.github/ISSUE_TEMPLATE/release.md

```diff
@@ -8,88 +8,98 @@ assignees: ''
 
 # Open Match Release Process
 
-Follow these instructions to create an Open Match release.  The output of the
+Follow these instructions to create an Open Match release. The output of the
 release process is new images and new configuration.
 
 ## Getting setup
 
-*note: the commands below are pasted from the 0.5 release.  make the necessary
-changes to match your naming & environment.*
+**NOTE: The instructions below are NOT strictly copy-pastable and assume 0.5**
+**release. Please update the version number for your commands.**
 
 The Git flow for pushing a new release is similar to the development process
 but there are some small differences.
 
-**1. Clone your fork of the Open Match repository.**
+### 1. Clone Repository
 
 ```shell
+# Clone your fork of the Open Match repository.
 git clone git@github.com:afeddersen/open-match.git
-```
-**2. Move into the new open-match directory.**
-
-```shell
+# Change directory to the git repository.
 cd open-match
-```
-
-**3. Configure a remote that points to the upstream repository. This is required to sync changes you make in a fork with the original repository.  Note: Upstream is the gatekeeper of the project or the source of truth to which you wish to contribute.**
-
-```shell
+# Add a remote, you'll be pushing to this.
 git remote add upstream https://github.com/googleforgames/open-match.git
 ```
 
-**3. Fetch the branches and their respective commits from the upstream repo.**
+### 2. Release Branch
+
+If you're creating the first release of the version, that would be `0.5.0-rc.1`
+then you'll need to create the release branch.
 
 ```shell
-git fetch upstream
+# Create a local release branch.
+git checkout -b release-0.5 upstream/master
+# Push the branch upstream.
+git push upstream release-0.5
```

Comment:
This won't work because it's protected now.

Suggested:
```
# Ensure using latest commit.
git fetch upstream
# Create a local release branch
git checkout -b release-0.5 upstream/master
# Push branch to personal github
git push --set-upstream origin release-0.5
```